python/src/Binomial_doc.i.in

```diff
@@ -1,16 +1,18 @@
 %feature("docstring") OT::Binomial
 "Binomial distribution.
 
-Available constructors:
-    Binomial(*n=1, p=0.5*)
-
 Parameters
 ----------
 n : int, :math:`n \in \Nset`
     The number of Bernoulli trials.
+
```

Comment:
One single blank line suffices to get the rendering you want.
```suggestion
```

Reply:
yes